etcdctl/command/doc.go

```diff
@@ -12,8 +12,5 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
-Package httptypes defines how etcd's HTTP API entities are serialized to and deserialized from JSON.
-*/
-
-package httptypes
+// Package command is a set of libraries for etcdctl commands.
+package command
```

Comment:
If you look at https://github.com/coreos/etcd/tree/master/etcdctl/command, it has multiple `*.go` files, so I think it's easier if we have a separate `doc.go` file for this directory.